domains/karan.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "LegendOP1041",
+           "email": "maillegendop@gmail.com",
+           "discord": "1056531806763102218"
+        },
+    
+        "record": {
+            "A": ["69.197.135.204"]
```

Comment:
Since you are using DBH proxy, i cant see the website. Could you please give me a preview of your website?